portfolio/src/main/webapp/script.js

```diff
@@ -123,3 +123,13 @@ function showSlides(n) {
   //the dot becomes 'active' (indicator is brighter)
   dots[slideIndex-1].className += " active";
 }
+
+/**
+ * Fetch data from the server.
+ */
+function getServerData() {
+  fetch('/data').then(response => response.text()).then((myJSON) => {
```

Comment:
Can you seperate this out onto different lines so it's a bit clearer to follow where the brackets close and open
eg:
// keeping the second then in line with the first, since it's on the same "level"
fetch().then()
            .then()

or

// indenting the then by 4 spaces since it's part of the above section.
fetch()
    .then()
    .then()

Reply:
ok apparently github strips the whitespace at the start of comment lines but hopefully the comments above the examples explain it.

Reply:
Ohh so indeting next line as well, I just put them on the same level but next line from your example

Reply:
Ok fixed it now, I hope it's what you meant.